pkg/pool-manager/pool.go

```diff
@@ -212,3 +213,16 @@ func (p *PoolManager) isInstanceOptedIn(namespaceName, mutatingWebhookConfigName
 
 	return false, nil
 }
+
+func GetMacPoolSize(rangeStart, rangeEnd net.HardwareAddr) (uint64, error) {
+	err := checkRange(rangeStart, rangeEnd)
+	if err != nil {
+		return 0, errors.Wrap(err, "mac Pool Size  is negative")
+	}
+
+	//making sure Uint64 receives an byte slice of size 8
+	startInt := binary.BigEndian.Uint64(append([]byte{0x0, 0x0}, rangeStart...))
```

Comment:
How about converting `rangeStart` and `rangEnd` to integers then calculate the range length,
that way you dont need mess with binary representation, padding, Endians etc...

Reply:
That's basically what I did :) 
rangeStart/End are basically of type []byte. I found [this](https://stackoverflow.com/questions/11184336/how-to-convert-from-byte-to-int-in-go-programming) to convert it to a interger, but am open to other suggestions =)

Reply:
Look at this example form shading branch
https://github.com/k8snetworkplumbingwg/kubemacpool/pull/122/commits/1972f47a956d57049942092feda7e363299af1a5#diff-baa41b07e1d50dce50eab577f6b940d5R125

Reply:
Like!